api/rs/slint/mcu.md

```diff
@@ -13,36 +13,26 @@ Writing an application in Rust that runs on a MCU requires several prerequisites
 * Install tools for flashing and debugging your code on the device.
 
 We recommend reading the [Rust Embedded Book](https://docs.rust-embedded.org/book/),
-as well as the curated list of [Awesome Embedded Rust](https://github.com/rust-embedded/awesome-embedded-rust) for a wide range of different
-crates, tools and training materials. These resources should guide you through the initial setup and often come with "hello world" examples
+and the curated list of [Awesome Embedded Rust](https://github.com/rust-embedded/awesome-embedded-rust) for a wide range of
+crates, tools and training materials. These resources should guide you through the initial setup. Many include a "hello world" example
 to get started with your device.
 
-Slint requires a global memory allocator. That is currently only possible in the nightly version of Rust, because the support for using a custom global
-allocator in a bare metal environment with `#![no_std]` has not been stabilized yet (see [#51540](https://github.com/rust-lang/rust/issues/51540) or
-[#66741](https://github.com/rust-lang/rust/issues/66741) for tracking issues).
+Slint requires a custom global memory allocator in a bare metal environment with `#![no_std]`.
+The rust team hasn't yet stabilized this feature, so only the nightly version
+of the Rust compiler will support it. See
+[#51540](https://github.com/rust-lang/rust/issues/51540) or
+[#66741](https://github.com/rust-lang/rust/issues/66741) for tracking issues.
 
 The following sections assume that your setup is complete and you have a non-graphical skeleton Rust program running on your MCU.
 
 ## Changes to `Cargo.toml`
 
-Start by adding a dependency to the `slint` and the `slint-build` crates to your `Cargo.toml`:
+Start by adding a dependency to the `slint` and the `slint-build` crates to your `Cargo.toml` using the `cargo` command:
 
-```toml
-[package]
-## ...
-## Edition 2021 or later enables the feature resolver version 2.
-edition = "2021"
+Start with the `slint` crate like this:
 
-[dependencies]
-## ... your other dependencies
-
-[dependencies.slint]
-version = "0.3.4"
-default-features = false
-features = ["compat-0-3-0", "unsafe-single-threaded", "libm"]
-
-[build-dependencies]
-slint-build = "0.3.4"
+```sh
+cargo add slint@1.0.0 --no-default-features --features "compat-1-0-0 unsafe-single-threaded libm"
```

Comment:
i have mixed feelings about that. I think it is usefull to show the Cargo.toml
Maybe we can have both.

Note that this should say that the use of edition 2021 is mandatory.
Also it is missing slint-build 
Note that the other PR calls the `compat-1-0`  without the last 0

Reply:
I agree. I see a future where editing `Cargo.toml` will not be necessary anymore, but for the time being it's still the common thing.

Another argument: That `cargo add` line is great for a one-time use, when you already know what features you need. But after that as a new user I'd be lost. "Ok, they have this magic line of features, but how do I add more after that?". Showing the structure of `Cargo.toml` solves that, because then the user knows where to make further edits.

I'm strongly in favor of keeping `Cargo.toml` for now.